optuna/visualization/_optimization_history.py

```diff
@@ -20,10 +24,11 @@
 
 
 def plot_optimization_history(
-    study: Study,
+    study: Union[Study, Sequence[Study]],
```

Comment:
I'm not sure whether it allows `study` to be a list of `Study`.

I approved this interface in https://github.com/optuna/optuna/pull/2103...
https://github.com/optuna/optuna/blob/e0fd830471cde878e2882c2ba1adaed307edd7c9/optuna/visualization/matplotlib/_edf.py#L28-L33

But I found it to be inconsistent with `direction` / `directions` style in: https://github.com/optuna/optuna/blob/e0fd830471cde878e2882c2ba1adaed307edd7c9/optuna/study/study.py#L1022-L1031

What do you think about it? Because `plot_edf` is experimental, it's time to consider a design.

Reply:
Personally, the current interface looks better than `direction` / `directions` since we can use the same interface not depending on the number of studies.

Reply:
Thank you, both are OK for me. I'd also like hear @hvy's opinion as you suggested to separate `directions` from `direction`.
https://github.com/optuna/optuna/pull/1994#discussion_r523928179

Reply:
Let me ping @hvy. 👀

Reply:
@himkt @nzw0301 Thanks for your comments. I prefer the current style, but it is not a strong opinion.

@hvy Do you have any comments?

Reply:
Personally, I'd target renaming this parameter to `studies`. I think that's part of the idea for the introduction of `directions`, although it might not be realized anytime soon. This is not a strong opinion however and the current design looks fine to me.

It's probably worth emphasizing that few users likely pass this argument with a keyword, i.e. you just assume that the first argument is the study (or studies). Having to make the argument a keyword argument when changing from a single study to multiple studies seems like poor design.